engine/core/src/main/java/org/datacleaner/job/builder/AnalysisJobBuilder.java

```diff
@@ -1000,7 +1014,7 @@ public void setDefaultRequirement(final ComponentRequirement defaultRequirement)
     /**
      * Gets a default requirement, which will be applied to all newly added row
      * processing components.
-     * 
+     *
```

Comment:
They look perfectly reasonable to me, why would there be a space after the *? Do you add spaces to lines between two method scopes?